### ISSUE TITLE
Interval time in different field

### ISSUE THREAD

**Issue by Paul-Reed** [bartbutenaers/node-red-contrib-interval-length#10]
Using current npm version - v0.0.3
The node readme describes how to move the interval time to a different field;

![image](https://user-images.githubusercontent.com/973580/60688223-c00a2f80-9eab-11e9-9b9f-0a83c3370d99.png)
But I'm not seeing a `Msg field` box in the node;

![image](https://user-images.githubusercontent.com/973580/60688324-65bd9e80-9eac-11e9-9450-4fec8cb8e466.png)

Also, ticking the `Add measurement in 'interval' field` in the node settings doesn't persist after deployment.

**Comment by bartbutenaers** [comment]
Evening Paul,

Long time ago that I wrote that code, so will need to have a better look at it tomorrow evening.
+ Saw quickly that I have changed the '_**Msg field**_' label to '_**Interval**_' in the code:
   ```
   <label for="node-input-typed-msgField"><i class="fa fa-list"></i> Interval</label>
   ```
   But the screenshot hasn't been updated ...

+ And seems I don't use that checkbox anymore, except in the html part.  Will need to remove that from the screen.

Summarized: I have been a bit a bit too hastily at the time being.  Sorry for that, and thanks for reporting!

Bart

**Comment by bartbutenaers** [comment]
Hi Paul (@Paul-Reed ),

About your question of 'yesterday' ...
Had completely forgotten your issue, until somebody else reminded me today.
I have published version **0.0.4** on NPM with following changes:
+ The checkbox *"add measurements in interval field"* has been removed, since that wasn't used anymore.
+ The screenshot has been changed on the readme page, to show *"Interval"* instead of *"Msg field"*.
+ A demo animation has been added on the readme to show how the Interval field works.

Bart
